src/docfx/build/Build.cs

```diff
@@ -26,14 +26,13 @@ internal static class Build
 
             var tocMap = await BuildTableOfContents.BuildTocMap(context, globbedFiles);
 
-            var (builtDocs, sourceDependencies) = await BuildFiles(context, globbedFiles, tocMap);
+            var (publishedFiles, sourceDependencies) = await BuildFiles(context, globbedFiles, tocMap);
 
-            BuildManifest.Build(context, builtDocs, sourceDependencies);
+            BuildManifest.Build(context, publishedFiles, sourceDependencies);
```

Comment:
>publishedFiles [](start = 41, length = 14)

build doesn't means publish

Reply:
+1, why do we name as "publish"

---
In reply to: [194339241](https://github.com/dotnet/docfx/pull/2868#discussion_r194339241) [](ancestors = 194339241)

Reply:
reverted

---
In reply to: [194599892](https://github.com/dotnet/docfx/pull/2868#discussion_r194599892) [](ancestors = 194599892,194339241)